tests/componenttests/client/tests/webaudio/CreateWebAudioPlayerTest.cpp

```diff
@@ -54,7 +54,16 @@ class CreateWebAudioPlayerTest : public ClientComponentTest
  *   Expect that  web audio api is called on the server
  *   Check that the object returned is valid.
```

Comment:
Need to add the client check here, like we do for MediaPipeline test

Reply:
This check is for web audio player client, and you also need to add a new test method to check this, same as we do for media pipeline MediaPipelineTestMethods::checkMediaPipelineClient();